src/EditorFactory.js

```diff
@@ -122,7 +122,7 @@ const createEditor = ({ content, onCreate, onUpdate, extensions, enableRichEditi
 	}
 	extensions = extensions || []
 	return new Editor({
-		content,
+		content: content + '<p/>',
```

Comment:
I fail to come up with a nicer approach for this, any ideas or objections on using this @nextcloud/text ?

Reply:
I can't think about better solution than this.